imap_processing/codice/codice_l1a.py

```diff
@@ -206,6 +191,99 @@ def get_acquisition_times(self):
             row_number = np.argmax(energy_steps == str(step_number), axis=1).argmax()
             self.acquisition_times.append(lo_stepping_values.acq_time[row_number])
 
+    def get_data_products(self, apid: int):
+        """Retrieve the values for the lo- or hi- data products table.
+
+        Parameters
+        ----------
+        apid : int
+            The APID of interest.
+        """
+        # TODO: For hi products, not sure what data array size should be yet,
+        # so just use 128 for now
+
+        data_product_configurations = {
```

Comment:
Nice style here for the lookups. Would it make sense to put this in the `constants` file as well alongside your other lookup tables?

Reply:
Yes, good idea!